src/plugins/RepairMan.h

```diff
@@ -10,5 +10,12 @@
 struct RepairMan : Plugin {
     void OnStep(Builder* builder_) final;
 
+    void OnUnitDamaged(const sc2::Unit* unit_, float health_, float shields_, Builder*) final;
+
     void OnUnitDestroyed(const sc2::Unit* unit_, Builder* builder_) final;
+
+    bool AssignRepairTask(const sc2::Unit& damaged_);
+
+ private:
+    std::list<const sc2::Unit*> m_damaged_buildings;
```

Comment:
Style: Please, include the corresponding header.

Reply:
What do you mean here? I don't understand what "corresponding header" refers to about line 20.

for std::list , I should include a list.h ?
for Unit*, I should include a sc2 unit.h ?

Reply:
Please provide more info here. I am confused.

Reply:
According to the Google styleguide, we should include std headers (<list> here) explicitly, even if the required header is available from another header.